### PR TITLE
Community Governance 

### PR DESCRIPTION
**Pull request by moonmeister**
## Description

Introduce a rough governance doc for initial community management. 

### Documentation

[Rendered Governance Doc](https://github.com/gatsby-uc/plugins/blob/community-mgmt/GOVERNANCE.md)

### CODE REVIEW

**Comment by moonmeister** [comment]
@LekoArts @pragmaticpat Thoughts? Also, I'd like to make one of you, or someone at Gatsby, a Steward as to reduce the bus factor. One of you okay with that or should it be someone like @KyleAMathews?

**Comment by LekoArts** [comment]
Thanks for putting this up! I'll give it a review tomorrow :)

**Comment by Burnsy** [comment]
LGTM! 👍🏼

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 Welcome! Submit a PR, answer questions, join the [Discord](https://discord.gg/gwnqfzetjv).
 
+We have a monthly community meeting that is listed in the Discord](https://discord.gg/gwnqfzetjv) server under events. The meeting is handled there in a voice channel. Anyone is welcome to come, listen, and join the conversation. The meeting is usually held on the last Tuesday of the month.
```
Comment (LekoArts) [inline CONTRIBUTING.md:5]:
```suggestion
We have a monthly community meeting that is listed in the [Discord](https://discord.gg/gwnqfzetjv) server under events. The meeting is handled there in a voice channel. Anyone is welcome to come, listen, and join the conversation. The meeting is usually held on the last Tuesday of the month.
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -64,7 +66,7 @@ Maybe this will get automated so all PR authors/contributors get added per plugi
 
 ### To build or not to build
 
-Not all packages NEED to be run through Babel. Any project with a non-insignificant amount of code should most likely be transpiled for compatability reasons. `babel-preset-gatsby-package` makes this easy but this does add a slight bit of maintenance over head. I'd default to adding transpilation to a package over not. Here is the minimal config for transpiling that a package will need.
+Not all packages NEED to be run through Babel. Any project with a non-insignificant amount of code should most likely be transpiled for compatability reasons. `babel-preset-gatsby-package` makes this easy but this does add a slight bit of maintenance over head. Here is the minimal config for transpiling that a package will need.
```
Comment (LekoArts) [inline CONTRIBUTING.md:69]:
```suggestion
Not all packages **need** to be run through Babel. Any project with a non-insignificant amount of code should most likely be transpiled for compatability reasons. `babel-preset-gatsby-package` makes this easy but this does add a slight bit of maintenance overhead. Here is the minimal config for transpiling that a package will need.
```

Capslock is not great on screenreaders 👂

File: GOVERNANCE.md [hunk 3]
```diff
@@ -0,0 +1,177 @@
+> Note: OUr model is extremely new, and not yet binding. Once merged into the `main` branch, we will trial what's outlined here and make changes based on any feedback. All changes will go through the PR review process. This note will be removed when this governance model is seen as binding.
```
Comment (LekoArts) [inline GOVERNANCE.md:1]:
```suggestion
> Note: Our model is extremely new, and not yet binding. Once merged into the `main` branch, we will trial what's outlined here and make changes based on any feedback. All changes will go through the PR review process. This note will be removed when this governance model is seen as binding.
```

File: GOVERNANCE.md [hunk 4]
```diff
@@ -0,0 +1,177 @@
...
+- **GitHub:** Filing a detailed bug report or RFC
+- **GitHub:** Updating documentation!
+- Helping people on GitHub, Discord, etc.
+- Answering questions on Stack Overflow, Twitter, etc.
+- Blogging, Vlogging, Podcasting, and Livestreaming about GUC projects
+- This list is incomplete! Similar contributions are also recognized.
+
+#### Privileges
+
+At this time we have no specifics for this role. as the community grows we anticipate privleges to come. If you have ideas here please let us know!
```
Comment (LekoArts) [inline GOVERNANCE.md:42]:
```suggestion
At this time we have no specifics for this role. As the community grows we anticipate privileges to come. If you have ideas here please let us know!
```

File: GOVERNANCE.md [hunk 5]
```diff
@@ -0,0 +1,177 @@
...
+
+This role does not require any extra responsibilities or time commitment. We hope you stick around and keep participating!
+
+If you're interested in reaching the next level and becoming a Maintainer, you can begin to explore some of those responsibilities in the next section.
+
+#### Nomination Process
+
+n/a
+
+### Level 2 (L2) - Maintainer
```
Comment (LekoArts) [inline GOVERNANCE.md:54]: Should Level 1 be (L1) then, too? (In the heading)

File: GOVERNANCE.md [hunk 6]
```diff
@@ -0,0 +1,177 @@
...
+
+The **Maintainer** role is available to any contributor who wants to join the team and take part in the long-term maintenance of a plugin(s) in the GUC.
```
Comment (LekoArts) [inline GOVERNANCE.md:56]:
```suggestion
The **Maintainer** role is available to any contributor who wants to join the team and take part in the long-term maintenance of plugins in the GUC.
```

From my standpoint it's also clear this way that it doesn't necessarily mean you maintain all plugins

File: GOVERNANCE.md [hunk 7]
```diff
@@ -0,0 +1,177 @@
...
+- **Discord:** Active participation in calls and other events
+- **GitHub + Discord:** Triaging and confirming user issues
+- This list is incomplete! Similar contributions are also recognized.
+
+#### Privileges
+
+- All privileges of the [Contributor role](#level-1---contributor), plus...
+- `@maintainer` role on [Discord](https://discord.gg/gwnqfzetjv)
+- New name color on Discord: **blue**.
+- Invitation to the private #maintainers channel on Discord.
```
Comment (LekoArts) [inline GOVERNANCE.md:79]:
```suggestion
- Invitation to the private `#maintainers` channel on Discord.
```

@pragmaticpat we want us (Gatsby) commit ourselves to also be in that channel and help out as needed. As in that sentence would also say: "[...] where also the Gatsby team can be reached"

File: GOVERNANCE.md [hunk 8]
```diff
@@ -0,0 +1,177 @@
...
+- Invitation to the `maintainers` team on GitHub.
+- Ability to moderate Discord.
+- Ability to push branches to the repo (No more personal fork needed).
+- Ability to review GitHub PRs.
+- Ability to merge _some_ GitHub PRs.
```
Comment (LekoArts) [inline GOVERNANCE.md:84]: What's "some"?

File: GOVERNANCE.md [hunk 9]
```diff
@@ -0,0 +1,177 @@
...
+
+The project Steward is currently: **@moonmeister**
+
+#### Responsibilities
+
+- Access to the [@gatsby_uc Twitter account](https://twitter.com/gatsby_uc)
+- Administration privileges on the [GUC GitHub org](https://github.com/gatsby_uc)
+- Administration privileges on the [GUC Discord server](https://discord.gg/GwNqFzeTJv)
+- Publish access to all GUC npm packages
+- Define project direction and planning
```
Comment (LekoArts) [inline GOVERNANCE.md:119]: I'd say that L2 members can weigh in on that, too?

**Review by LekoArts — CHANGES_REQUESTED** [review]
Thanks for that! Looks already great, left some comments :)

**Comment by pragmaticpat** [comment]
Thanks @moonmeister ! - I'll look at this one later on today / this evening. 👍

File: GOVERNANCE.md (hunk 8 above)
Comment (moonmeister) [reply GOVERNANCE.md:84]: This is leftover from astro. I thought I'd leave it. ATM they could merge anything.

File: GOVERNANCE.md [hunk 10]
```diff
@@ -0,0 +1,177 @@
...
+
+#### Nomination Process
+
+n/a
+
+### Level 2 - Maintainer
+
+The **Maintainer** role is available to any contributor who wants to join the team and take part in the long-term maintenance of plugins in the GUC.
+
+The Maintainer role is critical to the long-term health of GUC plugins. Maintainers act as the first line of defense when it comes to new issues, pull requests and Discord activity. Maintainers are most likely the first people that a user will interact with on Discord or GitHub.
```
Comment (pragmaticpat) [inline GOVERNANCE.md:58]:
```suggestion
The Maintainer role is critical to the long-term health of GUC plugins. Maintainers act as the first line of support when it comes to new issues, pull requests and Discord activity. Maintainers are most likely the first people that a user will interact with on Discord or GitHub.
```

File: GOVERNANCE.md [hunk 11]
```diff
@@ -0,0 +1,177 @@
...
+- Invitation to the `maintainers` team on GitHub
+- Ability to moderate Discord
+- Ability to push branches to the repo (No more personal fork needed)
+- Ability to review GitHub PRs
+- Ability to merge GitHub PRs
+- Define project direction and planning
+
+#### Responsibilities
+
+- Participate in the project as a team player.
```
Comment (pragmaticpat) [inline GOVERNANCE.md:89]:
```suggestion
- Participate in the project as a team player
```

Comment (pragmaticpat) [reply GOVERNANCE.md:89]: The intention here is just for consistency - either a trailing `.` or no trailing `.` on list items

File: GOVERNANCE.md [hunk 12]
```diff
@@ -0,0 +1,177 @@
...
+
+The project Steward is currently: **@moonmeister**
+
+#### Responsibilities
+
+- Access to the [@gatsby_uc Twitter account](https://twitter.com/gatsby_uc)
+- Administration privileges on the [GUC GitHub org](https://github.com/gatsby_uc)
+- Administration privileges on the [GUC Discord server](https://discord.gg/GwNqFzeTJv)
+- Publish access to all GUC npm packages
+- Ability to decide on moderation decisions
```
Comment (pragmaticpat) [inline GOVERNANCE.md:120]:
```suggestion
- Final say on moderation decisions
```

**Review by pragmaticpat — COMMENTED** [review]
Thanks for assembling this. I really like the fact that you're deriving the initial governance from Astro's model. I'm also a fan of Astro's model :)

File: GOVERNANCE.md (hunk 7 above)
Comment (pragmaticpat) [reply GOVERNANCE.md]: @LekoArts - I don't see which sentence you're specifically referencing. That said, I agree that there can be a mention of a Gatsby team member that at least checks in on some frequency. We should probably consider some sense of "office hours" for supporting any questions that arise. 

How's that sound to you and @moonmeister ?

Comment (moonmeister) [reply GOVERNANCE.md]: Doesn't sound like that needs to be in the governance docs. If we want to mention this as a resource in the contributing docs that seems fine. Unless we want to give maintainers explicit access to the Gatsby team.

File: GOVERNANCE.md (hunk 10 above)
Comment (moonmeister) [reply GOVERNANCE.md:58]: I can understand not using the word defense. But given the Readme where we explicitly say we're not providing support this seems like a contradiction. Thoughts?

File: GOVERNANCE.md (hunk 7 above)
Comment (LekoArts) [reply GOVERNANCE.md]: > If we want to mention this as a resource in the contributing docs that seems fine.

👍

**Review by LekoArts — APPROVED** [review]

**Pull request merged**